hardhat.config.ts

```diff
@@ -43,10 +43,25 @@ const config: HardhatUserConfig = {
         count: 10,
       },
     },
+    localhostl1: {
```

Comment:
What are these for?

Reply:
using this for fetching "companion" deployment on a different chain. eg -
https://github.com/livepeer/arbitrum-lpt-bridge/blob/kk%2Fdeploy/deploy/L1/gateway.deploy.ts#L13

more about it in the [docs](https://github.com/wighawag/hardhat-deploy#companionnetworks)

Reply:
I was specifically referring to whether we needed localhostl1 and localhostl2 in addition to rinkeby and arbitrumRinkeby, but I suppose these networks are useful if you are ever developing against local nodes.

Reply:
Oh, yeah I was initially trying to setup a local arbitrum dev node + hardhat l1 node for testing.